test/functional/test_framework/authproxy.py

```diff
@@ -106,7 +106,7 @@ def __getattr__(self, name):
 
     def _request(self, method, path, postdata):
         '''
-        Do a HTTP request, with retry if we get disconnected (e.g. due to a timeout).
+        Do an HTTP request, with retry if we get disconnected (e.g. due to a timeout).
```

Comment:
this was right, 'a' before the next consonant

Reply:
Not really. It depends how you pronounce it.

A universe.
An umbrella.

Reply:
Yeah it's an. We pronounce HTTP by reading the letters one by one, "H", "T", "T", "P"; and "H" is pronounced "aitch" not "haitch". If you poke around docs online you'll find the most (all?) big tech companies stick to "an" here and it's the accepted style. Google does, Toshiba does, Microsoft does.